### PR TITLE
update form item css

### LINKED ISSUE #14772: form组件 高级搜索例子 显示错误提示时在ie所有浏览器下布局错乱
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/form-cn/#components-form-demo-advanced-search](https://ant.design/components/form-cn/#components-form-demo-advanced-search)

### Steps to reproduce
在输入框输入一个值，然后按退格键删除

### What is expected?
布局正常
image

### What is actually happening?
布局错误
image

| Environment | Info |
|---|---|
| antd | 3.13.2 |
| React | react16 |
| System | win10 |
| Browser | ie11 |

### PR DESCRIPTION
**Pull request by zombieJ**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

fix #14772

### What's the effect? (Optional if not new feature)

This will affect multiple line explain by reduce 1px

### Changelog description (Optional if not new feature)

> 1. Fix FormItem height not align in IE.
> 2. 修复 FormItem 在 IE 下高度不一致的问题。

### Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
Please notes that this PR will affect multiple line explain by reduce 1px. @afc163 @ztplz

File: components/form/style/index.less [hunk 1]
```diff
@@ -10,6 +10,11 @@
 @form-component-max-height: @input-height-lg;
 @form-feedback-icon-size: @font-size-base;
 @form-help-margin-top: (@form-component-height - @form-component-max-height) / 2 + 2px;
+// Extends additional 1px to fix precision issue.
+// https://github.com/ant-design/ant-design/issues/12803
+// https://github.com/ant-design/ant-design/issues/8220
+@form-explain-precision: 1px;
+@form-explain-height: floor(@font-size-base * @line-height-base);
```
Comment (afc163) [inline components/form/style/index.less]: 应该是 ceil?

Comment (afc163) [reply components/form/style/index.less]: 哦上面有个 @form-explain-precision

Comment (zombieJ) [reply components/form/style/index.less]: 嗯，抽了个 `@form-explain-precision` 下面算起来会方便些。

**Comment by zombieJ** [comment]
Pls help check on this @afc163 @ztplz

**Review by ztplz — APPROVED** [review]

**Comment by afc163** [comment]
close #11880

**Pull request merged**